public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderMenuTrigger.tsx

```diff
@@ -17,7 +17,7 @@ export const PanelHeaderMenuTrigger: FC<Props> = ({ children, ...divProps }) =>
 
   const onMenuToggle = useCallback(
     (event: MouseEvent<HTMLDivElement>) => {
-      if (!isClick(clickCoordinates, eventToClickCoordinates(event))) {
+      if (!isClick(clickCoordinates, { x: event.clientX, y: event.clientY })) {
```

Comment:
🤔 is there any special reason to don't use the `eventToClickCoordinates`? it seems with the refactor, it will give exactly the same data

Reply:
you're right! just a leftover from messing around